File: src/main/java/com/googongill/aditory/external/chatgpt/dto/ChatGptResponse.java
package com.googongill.aditory.external.chatgpt.dto;

import lombok.Builder;
import lombok.Getter;

import java.io.Serializable;
import java.time.LocalDate;
import java.util.List;

@Getter
@Builder
public class ChatGptResponse implements Serializable {
    private String id;
    private String object;
    private LocalDate created;
    private String model;
    private List<Choice> choices;
}

File: src/main/java/com/googongill/aditory/external/chatgpt/dto/ChatGptResponse.kt
package com.googongill.aditory.external.chatgpt.dto

import java.io.Serializable
import java.time.LocalDate

data class ChatGptResponse(
    val id: String,
    val `object`: String,
    val created: LocalDate,
    val model: String,
    val choices: List<Choice>
) : Serializable
